Makefile

```diff
@@ -1,7 +1,19 @@
 MAKEFLAGS += --no-print-directory
 COV_FLAGS :=
 SHELL:=/bin/bash
+NPROC:=$(shell python -c "import multiprocessing as mp; print(mp.cpu_count())")
+
+# Build variables
+SR_BUILD := Release
+SR_LINK := Shared
+SR_TEST_REDIS_MODE := Clustered
+SR_TEST_RAI_VER := 1.2.7
+SR_WLM := Local
+SR_WLM_FLAGS :=
 SR_DEVICE := cpu
+SR_PEDANTIC := OFF
+SR_FORTRAN := OFF
```

Comment:
Also might want to surface the `BUILD_PYTHON` variable as well here to bypass the build of the python library